### DISCUSSION TITLE
DNS Issue with custom domain

### DISCUSSION THREAD

**Discussion by a1484414286** [community/community#43264] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I was trying to access my website but I received NET::ERR_CERT_COMMON_NAME_INVALID error.
I believe it's the issue with my SSL certificate, but I don't know how to solve it

**Comment by DSDmark** [comment, marked as answer]
Hi @a1484414286,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** or **link your website** which you were facing 👍.

**Reply by a1484414286** [reply]
It was my fault, I kept remove and save my custom domain, thich caused some DNS issue or some SSL certificate unmatch issue, now I know there's a re-run button under action... thank you tho!

**Comment by NickiUnicorn** [comment]
Hi! I have a similar problem, as I am trying to connect my Gandi domain to my Github. I have waited for a long time, but it keeps saying DNS in progress, or that it doesn't work. I have lowered my TTL on Gandi to 600 to make the prorcess faster. I have tried a multiple times and waited 10-20 hours, but it doesn't work. Do you possibly have an idea what the problem could be? :) Best wishes, Emma
![GANDI](https://user-images.githubusercontent.com/4794
![GITHUB](https://user-images.githubusercontent.com/47940414/212885538-6cf7c4b3-32b5-433e-a666-8edba9b29ebc.jpg)
0414/212885490-06dfcd48-66f8-446d-b788-9185434cba12.jpg)

**Comment by NickiUnicorn** [comment]
![GANDI](https://user-images.githubusercontent.com/47940414/212885744-df6fd96f-a5ad-415e-99d6-9c423c75597c.jpg)
